gax/src/main/java/com/google/api/gax/rpc/ClientContext.java

```diff
@@ -60,6 +61,7 @@
  */
 @AutoValue
 public abstract class ClientContext {
+  private static final String QUOTA_PROJECT_ID_HEADER_KEY = "x-goog-user-project";
```

Comment:
@bshaffer double check if the key is correct, looks like google miss `le`, but I saw others are also `goog`

Reply:
This is correct!